aiida/cmdline/commands/cmd_setup.py

```diff
@@ -18,8 +20,11 @@
 from aiida.manage.configuration import load_profile
 from aiida.manage.manager import get_manager
 
+CLI_LOGGER = logging.getLogger('cli')
+
 
 @verdi.command('setup')
+@options.VERBOSITY()
```

Comment:
No longer have to add this manually, no? Actually isn't this now added twice? Apparently nothing happens in that case?

Reply:
Good catch. Yes, it seems there is no issue with specifying it twice (it does not take up a function argument, everything is handled in the context)